### PR TITLE
report avg cpu and mem allocations for compute nodes across the cluster

### PR DESCRIPTION
**Pull request by joelddiaz**

### CODE REVIEW

File: ansible/roles/os_zabbix/vars/template_openshift_master.yml [hunk 1]
```diff
@@ -15,6 +15,20 @@ g_template_openshift_master:
     applications:
     - Openshift Master
 
+  - key: openshift.master.cluster.average.mem
```
Comment (twiest) [inline ansible/roles/os_zabbix/vars/template_openshift_master.yml]: consider `openshift.master.cluster.compute_nodes.allocated.mem.average` instead

File: ansible/roles/os_zabbix/vars/template_openshift_master.yml [hunk 2]
```diff
@@ -15,6 +15,20 @@ g_template_openshift_master:
...
+    description: "Average of memory percentage allocated across compute nodes"
+    value_type: int
+    units: "%"
+    applications:
+    - Openshift Master
+
+  - key: openshift.master.cluster.average.cpu
```
Comment (twiest) [inline ansible/roles/os_zabbix/vars/template_openshift_master.yml]: consider `openshift.master.cluster.compute_nodes.allocated.cpu.average` instead

File: scripts/monitoring/cron-send-cluster-capacity.py [hunk 3]
```diff
@@ -92,6 +92,10 @@ def load_nodes(self):
             if not node_ready:
                 continue
 
+            # Skip unschedulable nodes
+            if new_node['spec'].get('unschedulable'):
```
Comment (twiest) [inline scripts/monitoring/cron-send-cluster-capacity.py]: check for 'True'

File: scripts/monitoring/cron-send-cluster-capacity.py [hunk 4]
```diff
@@ -240,32 +244,69 @@ def how_many_schedulable(self, size):
...
+
+        return cluster_avg
+
+    def cluster_capacity(self):
+        ''' check capacity of compute nodes on cluster'''
 
         zbx_key_mem_alloc = "openshift.master.cluster.memory_allocation"
         zbx_key_max_pods = "openshift.master.cluster.max_mem_pods_schedulable"
+        zbx_key_avg_mem = "openshift.master.cluster.average.mem"
+        zbx_key_avg_cpu = "openshift.master.cluster.average.cpu"
```
Comment (twiest) [inline scripts/monitoring/cron-send-cluster-capacity.py]: update

File: scripts/monitoring/cron-send-cluster-capacity.py [hunk 5]
```diff
@@ -240,32 +244,69 @@ def how_many_schedulable(self, size):
...
 
         self.sql_conn = sqlite3.connect(':memory:')
 
         self.load_nodes()
         self.load_pods()
 
         memory_percentage = self.get_memory_percentage()
+        print "Percentage of memory allocated: {}".format(memory_percentage)
```
Comment (twiest) [inline scripts/monitoring/cron-send-cluster-capacity.py]: say "on compute nodes"

**Comment by twiest** [comment]
:+1:

**Comment by kwoodson** [comment]
:+1:

**Pull request merged**